src/main/java/net/sf/jabref/logic/importer/fetcher/GoogleScholar.java

```diff
@@ -36,7 +62,7 @@
             return pdfLink;
         }
 
-        String url = String.format(SEARCH_URL,
+        String url = String.format(SEARCH_IN_TITLE_URL,
```

Comment:
Could you please use the UriBuilder instead of replacing strings. It is more readable and you don't need to take care of manually encoding the input string.